exercises/rotational-cipher/.meta/generator/rotational_cipher_case.rb

```diff
@@ -0,0 +1,9 @@
+require 'generator/exercise_case'
+
+class RotationalCipherCase < Generator::ExerciseCase
+
+  def workload
+    assert_equal { "RotationalCipher.rotate(\"#{text.to_s}\", #{shiftKey.to_s})" }
```

Comment:
You need to put the `%Q` further out.

`assert_equal { %Q[RotationalCipher.rotate("#{text}", #{shiftKey})] }`

I've used square brackets as delimiters, this may or may not be a good idea.